index.html

```diff
@@ -6,13 +6,163 @@
+                    <i class="fa-solid fa-umbrella-beach "></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Recreation</h2>
+                    <p class="program-desc pt-2 font-light text-center text-sm md:text-base">Snacks would be distributed
+                        among audience and dinner would be served to them. kids are welcomed</p>
+                </div>
+                <div class="program-card gap-x-2  flex py-8 px-2 md:flex-col items-center justify-center">
+                    <i class="fa-solid fa-award"></i>
+                    <h2 class="text-lg md:text-xl program-subtitle pt-2">Entertainment</h2>
+                    <p class="program-desc pt-2 font-light text-sm md:text-base text-center">You will be
+                        eager to take autograph of your favorite actor/actress, so there is a meet and greet
+                        session</p>
+                </div>
+            </div>
+            <h3 class="text-center pt-8 text-2xl"><a href="#" class="underline  program-link">View full
+                    Program</a>
+            </h3>
+        </section>
+
+        <!-- speakers section -->
+
+        <section class="py-12 w-full speakers" id="speakers">
+            <h2 class="text-center text-3xl speakers-title pt-8">Featured Speakers</h2>
+            <div class="line"></div>
+            <div class="mt-12 grid md:grid-cols-2 gap-y-6 md:gap-x-4 mx-3 md:mx-10" id="speaker-container">
+            </div>
+        </section>
+
+        <!-- partners section -->
+        <section class="partners py-12" id="partners">
+            <h2 class="text-center text-3xl partners-title pt-8">Partners</h2>
+            <div class="line"></div>
+            <div class="w-full py-5 mt-5">
+                <div class="w-11/12 mx-auto">
+                    <div class="grid grid-cols-3 md:grid-cols-6 gap-4  justify-between ">
+                        <div class="bgImg">
+                            <img src="./images/client-1.png" class="img-fluid" alt="Responsive image">
+                        </div>
+                        <div class="bgImg">
+                            <img src="./images/client-2.png" class="img-fluid" alt="Responsive image">
+                        </div>
+                        <div class="bgImg">
+                            <img src="./images/client-3.png" class="img-fluid" alt="Responsive image">
+                        </div>
+                        <div class="bgImg">
+                            <img src="./images/client-6.png" class="img-fluid" alt="Responsive image">
+                        </div>
+                        <div class="bgImg">
+                            <img src="./images/client-4.png" class="img-fluid" alt="Responsive image">
+                        </div>
+                        <div class="bgImg">
+                            <img src="./images/client-5.png" class="img-fluid" alt="Responsive image">
+                        </div>
```

Comment:
- [ ] for best accessibility practices, kindly update the alt attributes for the partner's images to be more meaningful and unique 💯